models/entities/util.go

```diff
@@ -0,0 +1,5 @@
+package entities
+
+func as2(postfix string) string {
```

Comment:
This really needs a better name :x

Reply:
maybe put these functions in a ns package?

ns.AS2()
ns.LDP()

Reply:
Looks good! We could then use this in the typename constants as well.